addons/lunch/models/lunch_supplier.py

```diff
@@ -200,18 +200,29 @@ def _send_auto_email(self):
             'amount_total': sum(order.price for order in orders),
         }
 
+        sites = orders.mapped('user_id').mapped('last_lunch_location_id').sorted(lambda x: x.name)
+        orders_per_site = self.env['lunch.order']
+        for site in sites:
+            orders_per_site |= orders.filtered(lambda l: l.user_id.last_lunch_location_id == site)
+
         email_orders = [{
             'product': order.product_id.name,
             'note': order.note,
             'quantity': order.quantity,
             'price': order.price,
             'toppings': order.display_toppings,
             'username': order.user_id.name,
-        } for order in orders]
+            'site': order.user_id.last_lunch_location_id.name,
+        } for order in orders_per_site]
+
+        email_sites = [{
+            'name': site.name,
+            'address': site.address,
+        } for site in sites]
 
         self.env.ref('lunch.lunch_order_mail_supplier').with_context(
-            order=order, lines=email_orders
-        ).send_mail(self.id)
+            order=order, lines=email_orders, sites=email_sites
+        ).send_mail(self.id, force_send=True)
```

Comment:
Do not use force_send unless having good reasons to do so. You slowdown the whole transaction as people have to wait for the mail to be sent. You should always rely on the cron to send emails, as anyway you have only a few minutes of delay. Is it really necessary ?

Reply:
I +1 tbe, there are optimisations in the SaaS to send emails at a much higher rate (and `force_send` is ignored)

Reply:
Agreed, I thought the delay was slightly higher.